src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -96,10 +96,11 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
-        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
-        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
+        //Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        //Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        //return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        return new Person(updatedName, updatedPhone, updatedEmail, false);
```

Comment:
Does this mean that all edited person will have NOT be done?

What happens if I want to edit a previously done contact?

Reply:
Oh right I'll update this part

Reply:
Oh, edit is not a feature we keeping from the original? Its still being kept inside the Userguide iirc

Reply:
Nope not in the userguide

Reply:
Oh ok. Then should just leave it as that